ttnn/cpp/ttnn/operations/downsample/device/downsample_op.hpp

```diff
@@ -25,6 +28,21 @@ struct Downsample {
     const auto attribute_values() const {
         return std::make_tuple(std::cref(this->downsample_params), std::cref(this->output_dtype));
     }
```

Comment:
attribute_names and attribute_values can be removed
See https://github.com/tenstorrent/tt-metal/commit/a0dbfe07f6726421ea90eeb02fe3548b9e5fee83

Reply:
Done.